lambda_functions/build.py

```diff
@@ -84,7 +84,10 @@ def _build_downloader(target_directory: str) -> None:
         shutil.rmtree(temp_package_dir)
 
     # Pip install cbapi library (has no native dependencies).
-    pip.main(['install', '--quiet', '--target', temp_package_dir, 'cbapi==1.3.6'])
+    subprocess.check_call([
```

Comment:
Out of my curiosity. What's the reason that you are prefer to use `subprocess` over `pip`?

Reply:
The `pip` Python library was never officially supported. In the latest `pip` release (v10), they enforce this by removing the `pip.main` method entirely